docs/administration/product-feature-matrix.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 2
-title: Product Feature Matrix
+title: Product feature matrix
```

Comment:
I think a redirect is needed here. 
I searched "architecture" in the local deployment for this branch, and I am not directed to the <administration> tab.
![image](https://github.com/user-attachments/assets/2b215d80-f18e-40ed-87ff-32ad558f301b)